src/vuestic-theme/vuestic-components/va-date-picker/VaDatePicker.demo.vue

```diff
@@ -1,102 +1,141 @@
-<template>
-  <div class="demo-container">
-    <div class="demo-container__item">
-      <div class="form-group">
-        <div class="input-group">
-          <va-date-picker
-            id="simple-input"
-            v-model="simple"
-          />
-          <label class="control-label" for="simple-input">Simple</label>
-          <va-icon icon="bar"/>
-        </div>
-      </div>
-
-      {{ simple }}
-    </div>
-    <div class="demo-container__item">
-      <div class="form-group">
-        <div class="input-group">
-          <va-date-picker
-            id="simple-input-2"
-            v-model="range"
-            :config="{mode: 'range'}"
-          />
-          <label class="control-label" for="simple-input-2">Range</label>
-          <va-icon icon="bar"/>
-        </div>
-      </div>
-
-      {{ range }}
-    </div>
-    <div class="demo-container__item">
-      <div class="form-group">
-        <div class="input-group">
-          <va-date-picker
-            id="simple-input-4"
-            v-model="dayTime"
-            :config="{enableTime: true}"
-            @on-change="$vb.log('on-change', $event)"
-          />
-          <label class="control-label" for="simple-input-4">Day time</label>
-          <va-icon icon="bar"/>
-        </div>
-      </div>
-      <div class="form-group">
-        <div class="input-group">
-          <va-date-picker
-            id="simple-input-3"
-            v-model="dayTime"
-            :config="{enableTime: true}"
-          />
-          <label class="control-label" for="simple-input-3">Day time</label>
-          <va-icon icon="bar"/>
-        </div>
-      </div>
-      {{ dayTime }}
-    </div>
-    <div class="demo-container__item">
-      <div class="form-group">
-        <div class="input-group">
-          <va-date-picker
-            id="simple-input-5"
-            v-model="readableDayTime"
-            :config="{
-              altInput: true,
-              altFormat: 'F j, Y',
-              dateFormat: 'Y-m-d',
-            }"
-            @on-change="$vb.log('on-change', $event)"
-          />
-          <label class="control-label" for="simple-input-5">Human
-            readable</label>
-          <va-icon icon="bar"/>
-        </div>
-      </div>
-      {{ readableDayTime }}
-    </div>
-  </div>
-</template>
-
-<script>
-import VaDatePicker from './VaDatePicker.vue'
-
-export default {
-  components: { VaDatePicker },
-  data () {
-    return {
-      simple: null,
-      range: null,
-      dayTime: null,
-      four: null,
-      readableDayTime: null,
-    }
-  },
-  methods: {
-    doSomethingOnChange (input) {
-      // eslint-disable-next-line no-console
-      // console.log('onChange', input)
-    },
-  },
-}
-</script>
+<template>

```

Comment:
These arrowheads are very hard to hit.
![image](https://user-images.githubusercontent.com/5394573/55968528-dc1f7b80-5c84-11e9-93c6-1bee94952c41.png)